src/components/FullscreenGrid/styles.module.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../scss/variables';
+
+.FullscreenGrid {
+  display: grid;
+  grid-gap: 10px;
+  grid-template-areas:
+    'FGHeader FGHeader FGHeader'
+    'FGNavPanel FGContentPanel FGAltPanel'
+    '. . .';
+  grid-template-columns: 1fr 4fr 1fr;
+  grid-template-rows: min-content 1fr 0px;
+  height: 100vh;
+}
+
+.Header {
+  grid-area: FGHeader;
+}
+
+.NavPanel {
+  grid-area: FGNavPanel;
+  margin-left: 10px;
+}
+
+.AltPanel {
+  grid-area: FGAltPanel;
+  margin-right: 10px;
```

Comment:
I suppose we could introduce a variable for this `10px` value.

Reply:
Did you mean in the whole app or just this style module? I started with one just in this style module. That makes sense.

Reply:
I just added a module variable for now.